internal/impl/deploy.go

```diff
@@ -86,6 +88,16 @@ func Deploy(ctx context.Context, configFilename string) error {
 	if config.Repo == "" {
 		fmt.Fprintln(os.Stderr, "No container repo specified in the config file. The container image will only be accessible locally. See `weaver kube deploy --help` for details.")
 	}
+	switch strings.ToLower(config.BuildTool) {
```

Comment:
I think the user should specify either "docker" or "podman". We shouldn't allow "Docker", "Podman" or other variations, so let's not do the lowercasing.

If the config.BuildTool is already "docker" or "podman" we don't need to set it again. 

Please see my previous comment on how you can rewrite this.

Reply:
Done. :)